src/main/java/racingcar/ui/InputView.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.ui;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class InputView {
+    private static InputView instance = new InputView();
+
+    private InputView() {
+
+    }
+
+    public static InputView getInstance() {
+        return instance;
+    }
+
+    public void printNumberOfCar() {
+        System.out.println("자동차 대수는 몇 대 인가요?");
+    }
+
+    public void printNumberOfAttempt() {
+        System.out.println("시도할 회수는 몇 회 인가요?");
+    }
+
+    public static int inputNumberOfCar(Scanner scanner) {
+        InputView.getInstance().printNumberOfCar();
+        try {
+            return scanner.nextInt();
+        } catch (InputMismatchException e) {
+            System.out.println("정수 타입이 아닙니다!" + e);
+            throw new RuntimeException();
```

Comment:
적절한 표준 예외가 좋겠습니다. 
또한 출력보다는 사용자에게 어떤 문제로 예외가 발생했는지 메시지를 담아주는 것이 더 좋겠죠?
```suggestion
            throw new IllegalArgumentException(메시지);
```